rest/config.go

```diff
@@ -386,14 +387,23 @@ func (dbConfig *DbConfig) setup(name string) error {
 // If the specified path does not qualify for a valid file or an URI, it returns the input path
 // as-is with the assumption that it is an inline JavaScript source. Returns error if there is
 // any failure in reading the JavaScript file or URI.
-func loadJavaScript(path string) (js string, err error) {
-	rc, err := readFromPath(path)
+func loadJavaScript(path string, insecureSkipVerify bool) (js string, err error) {
+	rc, err := readFromPath(path, insecureSkipVerify)
 	if errors.Is(err, ErrPathNotFound) {
 		// If rc is nil and readFromPath returns no error, treat the
 		// the given path as an inline JavaScript and return it as-is.
 		return path, nil
 	}
 	if err != nil {
+		if !insecureSkipVerify {
+			var unkAuthErr x509.UnknownAuthorityError
+			if errors.As(err, &unkAuthErr) {
+				return "", fmt.Errorf("%w. SSL certificate verification is turned "+
+					"on by default when loading config over HTTPS. Can be overriden by "+
+					"the unsupported \"remote_config_tls_skip_verify\" option", err)
```

Comment:
```suggestion
				return "", fmt.Errorf("%w. TLS certificate failed verification.  TLS verification "+
					"can be disabled using the unsupported \"remote_config_tls_skip_verify\" option", err)
```

Reply:
This change has been committed in the latest revision.